core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -394,6 +395,7 @@ private void addCallbackAndFlush(Throwable cause, State oldState, ChannelFuture
                     }
                     // Write an access log always with a cause. Respect the first specified cause.
                     if (tryComplete()) {
+                        logBuilder().endRequest(cause);
```

Comment:
Question: Is it possible to automatically complete a request-side log with the `cause` when `logBuilder().endResponse(cause)` is called and `endRequest(...)` is not called?

Reply:
Yeah, I have thought about it but I'm not sure if it's a good idea because we usually handle it separately when the request and response are completed in both the client-side and server-side. So different cause might be set.
However, this is a special case in which the request is not subscribed at all at the server-side. So I thought we need to call `endRequest()` to make sure the log is complete
so that the order of all access logs are not mixed up.
Do you think it's better to call `endRequest()` in `endResponse()`?

Reply:
I wanted to prevent devs from making unexpected mistakes. But, either is fine with me.
Let's discuss later if we have a chance to consider a topic related to this.